src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -2768,6 +2768,52 @@ public override BoundNode VisitNonConstructorMethodBody(BoundNonConstructorMetho
             return null;
         }
 
+        public override BoundNode VisitNullCoalescingAssignmentOperator(BoundNullCoalescingAssignmentOperator node)
+        {
+            LocalState savedState;
+            if (RegularPropertyAccess(node.LeftOperand))
+            {
+                var left = (BoundPropertyAccess)node.LeftOperand;
+                var property = left.PropertySymbol;
+                if (property.RefKind == RefKind.None)
+                {
+                    var readMethod = property.GetOwnOrInheritedGetMethod() ?? property.GetMethod;
```

Comment:
> ?? property.GetMethod [](start = 74, length = 22)

`property.GetMethod` is null, otherwise `property.GetOwnOrInheritedGetMethod()` would have returned non-null. Same comment for `SetMethod`.

Reply:
We should just remove `?? property.GetMethod` and `?? property.SetMethod`.

---
In reply to: [201412511](https://github.com/dotnet/roslyn/pull/28253#discussion_r201412511) [](ancestors = 201412511)